pkg/minikube/machine/cache_images.go

```diff
@@ -64,12 +64,20 @@ func CacheImagesForBootstrapper(imageRepository string, version string, clusterB
 
 // LoadImages loads previously cached images into the container runtime
 func LoadImages(cc *config.ClusterConfig, runner command.Runner, images []string, cacheDir string) error {
-	// Skip loading images if images already exist
-	if cruntime.DockerImagesPreloaded(runner, images) {
-		glog.Infof("Images are preloaded, skipping loading")
-		return nil
+	if cc.KubernetesConfig.ContainerRuntime == "docker" {
```

Comment:
We could move these into a helper function as well that could be called throughout the code:

```go
func ImagesPreloaded(containerRuntime string) bool }
}
```

Reply:
good idea

Reply:
done